__tests__/app.test.js

```diff
@@ -115,3 +115,54 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id", () => {
+  test("returns status 200 and the updated restaurant object with updated votes amount when successful", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .expect(200)
+      .send({ inc_votes: -100 })
+      .then(({ body: article }) => {
+        expect(article.article).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 0
+          })
+        );
+      });
+  });
+  test("400: responds with an error when passed an article_id of an incorrect type", () => {
+    return request(app)
+      .patch("/api/articles/not-a-number")
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid ID type");
+      });
+  });
+  test("404: responds with an error when passed an article_id not present in our database", () => {
+    return request(app)
+      .patch("/api/articles/100000")
+      .send({ inc_votes: 1 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("article_id not found in the database");
+      });
+  });
+  test("400: responds with an error when patch request has no content for the update", () => {
```

Comment:
I would argue that this is not a bad request, all that we care about is whether the key exists and if it doesn't then this shouldn't error out our endpoint. This would mean that the absence of a specific key won't cause an error and allows us to be more flexible with future requests should we extend our endpoint to handle patches to other parts of the articles data. 

In this case I would assert that we would get a 200 and return the article unchanged.

Reply:
Additionally you should add a test for a bad request in the form of `{article_id: "notanumber"}`